### PR TITLE
fix: span names, log levels and messages

### PR DESCRIPTION
**Pull request by jonaro00**

### CODE REVIEW

**Review by chesedo — COMMENTED** [review]
Looks good

**Comment by jonaro00** [comment]
Missed a detail on colorize. We can probably remove tracing feature from codegen now because if the move.

File: runtime/src/alpha/mod.rs [hunk 1]
```diff
@@ -64,6 +64,43 @@ pub async fn start(loader: impl Loader<ProvisionerFactory> + Send + 'static) {
         }
     };
 
+    // this is handled after arg parsing to not interfere with --version above
+    if cfg!(feature = "setup-tracing") {
```
Comment (orhun) [inline runtime/src/alpha/mod.rs:68]: Why this was moved to here from codegen?

Comment (jonaro00) [reply runtime/src/alpha/mod.rs:68]: It is not needed in codegen, and it interfered with the arg parsing of runtime, which the comment states.

File: cargo-shuttle/src/lib.rs [hunk 2]
```diff
@@ -843,12 +843,9 @@ impl Shuttle {
             while let Ok(message) = rx.recv() {
                 match message {
                     Message::TextLine(line) => println!("{line}"),
-                    Message::CompilerMessage(message) => {
-                        if let Some(rendered) = message.message.rendered {
-                            println!("{rendered}");
-                        }
+                    message => {
+                        trace!("skipping cargo line: {message:?}")
```
Comment (oddgrd) [inline cargo-shuttle/src/lib.rs:847]: Is this to avoid the super verbose cargo messages we say during builds?

File: common/src/claims.rs [hunk 3]
```diff
@@ -219,7 +219,7 @@ impl Claim {
         let mut validation = Validation::new(jsonwebtoken::Algorithm::EdDSA);
         validation.set_issuer(&[ISS]);
 
-        trace!(token, "converting token to claim");
+        // trace!(token, "converting token to claim");
```
Comment (oddgrd) [inline common/src/claims.rs:222]:
```suggestion
        trace!("converting token to claim");
```

**Comment by iulianbarbu** [comment]
> Missed a detail on colorize. We can probably remove tracing feature from codegen now because of the move.

It might be readded because of `shuttle-next`. I am still working through the default tracing for it, blocked by some wasmtime related impl details.

File: deployer/src/deployment/run.rs [hunk 4]
```diff
@@ -423,13 +416,12 @@ async fn run(
...
         .into_inner();
 
-    info!("starting service");
     let response = runtime_client.start(start_request).await;
 
     match response {
         Ok(response) => {
             if response.into_inner().success {
-                info!(DEPLOYER_RUNTIME_START_RESPONSE);
+                info!("{}", DEPLOYER_RUNTIME_START_RESPONSE);
```
Comment (oddgrd) [inline deployer/src/deployment/run.rs:424]: Just curious, what is the benefit of this change?

**Review by oddgrd — COMMENTED** [review]
Looks good! Just left some questions.

File: cargo-shuttle/src/lib.rs (hunk 2 above)
Comment (jonaro00) [reply cargo-shuttle/src/lib.rs:847]: Yes

File: common/src/claims.rs (hunk 3 above)
Comment (jonaro00) [reply common/src/claims.rs:222]: @iulianbarbu How about this?

File: deployer/src/deployment/run.rs (hunk 4 above)
Comment (jonaro00) [reply deployer/src/deployment/run.rs:424]: Logging `string` vs logging `{VARIABLE_NAME="string"}`

**Review by iulianbarbu — APPROVED** [review]
LGTM! resolved my comments

**Pull request merged**